mantidproject/mantid
# Allow the efixed option in ISIS Energy Transfer to actually set the efixed

Fixes #13109 

To test:
- Open Indirect > Data Reduction > ISIS Energy Transfer
- Instrument: IRIS, graphite, 002
- Run: 26176
- Run, plot the contour plot
- Change Efixed to 1.9
- Run, elastic peak should shift to the negative energy
- Change Efixed to 1.7
- Run, elastic peak should shift to the positive energy
- See that when you change to an instrument other than IRIS and OSIRIS the option is disabled

Release notes updates [here](http://www.mantidproject.org/index.php?title=Release_Notes_3_5_Indirect_Inelastic&diff=24546&oldid=24523).
